src/reply_parser.c

```diff
@@ -0,0 +1,192 @@
+#include "reply_parser.h"
+#include "server.h"
+
+/* ----------------------------------------------------------------------------------------
+ * A resp parser used to parse replies returned by RM_Call
```

Comment:
nit: This comment block needs indentation and at least another round of review for spelling/grammar/etc. Also, please avoid tab characters.